README.md

```diff
@@ -381,27 +381,41 @@ I've picked `ATTRS{model}`.
 Let's combine what we've found into a shell script...
 
```

Comment:
The original format of the `README` was along the lines of a log/journal documenting the steps I took to probe and develop a solution.

I think it might be best to include a link to the files within the repository versus including a copy of the shell script and other assets. As the latter would necessitate the duplication of any changes made to the underlying files ...into the README.

Reply:
Updated accordingly.